src/components/card/CardSidebar.vue

```diff
@@ -170,11 +175,46 @@
 			icon="icon-activity">
 			<CardSidebarTabActivity :card="currentCard" />
 		</AppSidebarTab>
+		<Modal v-if="modalShow" :title="t('deck', 'Choose attachment')" @close="modalShow=false">
+			<div class="modal__content">
+				<h3>{{ t('deck', 'Choose attachment') }}</h3>
+				<Actions>
+					<ActionButton icon="icon-close" @click="modalShow=false">
+						{{ t('deck', 'Cancel') }}
+					</ActionButton>
+				</Actions>
+
+				<div class="attachment-list">
+					<ul>
+						<li v-for="attachment in attachments"
+							:key="attachment.id"
+							class="attachment">
+							<a class="fileicon" :style="mimetypeForAttachment(attachment.extendedData.mimetype)" :href="attachmentUrl(attachment)" />
```

Comment:
Please move that into a separate component so it can be reused. Right now this is just duplicating logic we already have in the attachments sidebar.